x-pack/plugins/maps/public/angular/get_initial_layers.js

```diff
@@ -15,26 +15,27 @@ export function getInitialLayers(savedMapLayerListJSON) {
     return JSON.parse(savedMapLayerListJSON);
   }
 
-  if (!isMetaDataLoaded()) {
-    const descriptor = EMSTMSSource.createDescriptor(DEFAULT_EMS_TILE_LAYER);
-    const source = new EMSTMSSource(descriptor);
+  const tilemapSourceFromKibana = getKibanaTileMap();
```

Comment:
that is great that by making `getKibanaTileMap` sync we can move the check above `isMetaDataLoaded `. This makes more logical sense